ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -0,0 +1,330 @@
+                                  </label>
+                                  <div style={{...styles.text, fontSize: 14}}>
+                                    This will be the primary email contact for your new account.
+                                  </div>
+                                </div>}
+                                invalid={this.state.emailFailedValidation}
+                                onBlur={() => this.validateContactEmailInline()}
+                                onChange={email => this.updateContactEmail(email)}/>
+            {this.state.emailFailedValidation &&
+              <ErrorDiv data-test-id='invalid-email-error'>
+                Error: email address is invalid
+              </ErrorDiv>
+            }
+            <div style={{marginTop: '.5rem'}}>
+              <label style={{...styles.text, fontWeight: 600, marginTop: '1rem'}}>
+                Which of the following best describes your role?
+                <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
+                  Publicly displayed
+                </i>
+              </label>
+              <div>
+                <Dropdown data-test-id='role-dropdown'
+                          style={{width: '50%', 'minWidth': '600px'}}
+                          placeholder={this.getRoleOptions() ?
+                            '' : 'First select an institution above'}
+                          options={this.getRoleOptions()}
+                          value={institutionalRoleEnum}
+                          onChange={(e) => this.updateAffiliationValue('institutionalRoleEnum', e.value)}/>
+              </div>
+            </div>
+            {institutionalRoleEnum === InstitutionalRole.OTHER && <div style={{marginTop: '.5rem'}}>
+              <label style={{...styles.text, fontWeight: 600, marginTop: '1rem'}}>
+                Please describe your role
+                <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
+                  Publicly displayed
+                </i>
+              </label>
+              <TextInputWithLabel value={institutionalRoleOtherText}
+                                  inputId='institutionalRoleOtherText'
+                                  inputName='institutionalRoleOtherText'
+                                  onChange={v => this.updateAffiliationValue('institutionalRoleOtherText', v)}/>
+            </div>
+            }
+          </div>
+          }
+          <FormSection style={{paddingBottom: '1rem'}}>
+            <Button type='secondary' style={{marginRight: '1rem'}}
+                    onClick={() => this.props.onPreviousClick(this.state.profile)}>
+              Previous
+            </Button>
+            <TooltipTrigger content={errors && <div data-test-id='validation-errors'>
+              <div>Please review the following: </div>
+              <ul>
+                {Object.keys(errors).map((key) => <li key={errors[key][0]}>{errors[key][0]}</li>)}
```

Comment:
TOL: I wonder if we should consider making this a helper function, since we use this pattern a decent amount.

Reply:
I think I'm happy not doing that in this PR, but would you add a ticket to follow up on that? (Maybe a tech debt ticket?)

Reply:
Sure, filed RW-4611